l10n_it_withholding_tax/models/withholding_tax.py

```diff
@@ -173,6 +183,10 @@ def _compute_total(self):
             statement.amount_paid = tot_wt_amount_paid
 
     date = fields.Date('Date')
+    type = fields.Selection([
```

Comment:
``type`` è una keyword riservata

Reply:
Vero però in odoo core ci sono diversi campi chiamati `type`, ad esempio in fattura. Dici che vale la pena preoccuparcene qui?

Reply:
Si ho visto che vengono usate, e secondo me è una brutta pratica perchè nasconde le keyword e può comportare a bug divertenti da debuggare. Personalmente eviterei di usare keyword riservate a prescindere.

Reply:
:+1: modificato